### PR TITLE
Pull 1 : add authorization routes(working), web-socket to listen frontend(logic broken)

### PR DESCRIPTION
**Pull request by Pragati1610**

### CODE REVIEW

File: Login/models/model.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+
+const userSchema = mongoose.Schema({
+    _id: mongoose.Schema.Types.ObjectId,
+    email: {
+        type: String, 
+        required: true, 
+        unique: true,
+    match: /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/ },
+    password: {type: String, required: true}
```
Comment (L04DB4L4NC3R) [inline Login/models/model.js:10]: Need a max and min limit of the password at the DB level

File: Login/routes/login.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+router.post("/signup", (req, res) => {
+    User.find({
+        email: req.body.email
+    }).exec().then(useri => {
+        if (useri.length >= 1) {
+            return res.status(409).json({
+                message: "mail exists"
+            });
+        } else {
+            bcrypt.hash(req.body.password, 10, (err, hash) => {
```
Comment (L04DB4L4NC3R) [inline Login/routes/login.js:18]: Hash salt is leaked here

File: app.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    //     if(err){
+    //         console.log(`Error: ${err.message}`);
+    //     }
+    //     if(stderr){
+    //         console.log(`Error: ${stderr}`);
+    //     }
+    //     console.log(stdout)
+    // });
+
+    exec("groff -i ms -T html >> out.html", (err,stdout,stderr)=>{
```
Comment (L04DB4L4NC3R) [inline app.js:59]: Dont `>> out.html`, we need it outputted to stdout, not a file

**Comment by L04DB4L4NC3R** [comment]
@Pragati1610 do the needful changes

**Comment by Pragati1610** [comment]
Create a new pull request

**Pull request closed without merging**